experiment/oss_fuzz_checkout.py

```diff
@@ -104,6 +104,10 @@ def postprocess_oss_fuzz() -> None:
   if os.path.exists(os.path.join(OSS_FUZZ_DIR, VENV_DIR)):
     return
 
+  # If already in a virtualenv environment assume all is set up
+  if os.environ['VIRTUAL_ENV']:
```

Comment:
Maybe `os.getenv('VIRTUAL_ENV')`?

Reply:
+1. this should exception otherwise right?

Reply:
Thanks! Fixed